config/database.yml

```diff
@@ -1,6 +1,11 @@
+# SQLite version 3.x
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem 'sqlite3'
 development:
-  adapter: postgresql
```

Comment:
Can't remember, this was around the time I was updating the Heroku stack to Cedar-14. I think I noticed we were using Postgres on production so I thought we might as well use it for dev too, but it could have also been related to some sqlite3 dependency error that I just wanted to get around quickly. Needless to say, there is no strong reason to use Postgres over sqlite3 for dev, so I think it's fine to switch it back here if it's working!